roxie/ccd/ccdmain.cpp

```diff
@@ -823,10 +822,18 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
         }
         directoryTree.clear();
 
+        // Tracing feature flags
+        setTraceFlag(loadTraceFlags(topology, roxieOptions, traceRoxieActiveQueries));
+
         //Logging stuff
 #ifndef _CONTAINERIZED
         if (topology->getPropBool("@stdlog", traceLevel != 0) || topology->getPropBool("@forceStdLog", false))
-            queryStderrLogMsgHandler()->setMessageFields(MSGFIELD_time | MSGFIELD_milliTime | MSGFIELD_thread | MSGFIELD_prefix);
+        {
+            if (topology->getPropBool("@minlog", false))
```

Comment:
Another change it would be useful to know if this is a new feature or a fix

Reply:
It allows me to output logging without the timestamps/thread IDs etc that might vary between runs, to make comparing the effects of changes to the logging code easier.